_data/accessibility-tests/icon.yml

```diff
@@ -0,0 +1,50 @@
+title: Icon
+component_status: pass
+test_items:
+# General items
+  - summary: Color is not the only method used to convey icon information.
+    summary_additional: |
+      When you view an icon that represents an action and determine the action,
```

Comment:
Suggest edit: "When you view an icon paired with, text, and never color alone, conveys the icon’s meaning."

Reply:
Updated in [abe962f](https://github.com/uswds/uswds-site/pull/2518/commits/abe962f72b0e14d17d23ecf9edbb4e406da5e570)